ros/src/computing/planning/motion/packages/astar_planner/nodes/velocity_set/velocity_set.cpp

```diff
@@ -649,6 +652,7 @@ int main(int argc, char** argv)
 
     int obstacle_waypoint = -1;
     double obstacle_velocity = 0.0;
+    current_velocity_ = vs_path.getCurrentVelocity();
```

Comment:
are we back to initialization from waypoints' velocity?

Reply:
this is ego car current velocity used to initialize the `prev_velocity_` variable.